book/configuration.md

```diff
@@ -113,4 +128,3 @@ you may want to configure `STARSHIP_SHELL` environment variable when setting the
 config set prompt "STARSHIP_SHELL=nushell starship prompt"
```

Comment:
config set prompt?

Reply:
I can delete that too. It's in the current book